_posts/2019-09-10-guide_building_the_game.md

```diff
@@ -48,15 +48,17 @@ It may also be called `origin/develop`:
 </div>
 
 ## Windows
-Download Momentum Mod on Steam **(see the important note at the top of this page).**
+Download Momentum Mod on Steam or grab the `MomentumDev` folder from [***Google Drive***](https://drive.google.com/file/d/115xT5IIN-CimKCZQho3xj7uIcYFMSZXm/view).
 
 Download [***Visual Studio 2019***](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=16).
 
 Follow [this step on the Valve Dev Wiki](https://developer.valvesoftware.com/wiki/Source_SDK_2013#Unable_to_find_RegKey_for_.vcproj_files_in_solutions_.28Windows.29) to add a missing registry key to your computer.
 
 Get a Git client like [***GitHub Desktop***](https://desktop.github.com/) if you haven't already and then clone this repository.
 
-Create an extra folder somewhere called `MomentumDev` and copy all of the files from your Steam's `steamapps/common/Momentum Mod` directory into this folder.
+If you downloaded Momentum Mod from Steam, create an extra folder somewhere called `MomentumDev`.
+
+Copy all of the files **except** the `momentum` folder from your Steam's `steamapps/common/Momentum Mod` directory into this folder.
```

Comment:
I wouldn't say to do this, as it makes the next line impossible to verify if it was copied correctly. It's a one-time copy anyways, saving people from copying then deleting it won't really do much other than make it confusing.

Reply:
I think it makes more sense because the MomentumDev in Google Drive doesn't include the momentum folder. I didn't want people to get confused like "Huh I can't delete this folder because I don't have it."

Reply:
I could rephrase it to "copy everything and then delete the momentum folder" or something, but I feel this is a better way to do it.

Reply:
Maybe condense line 59 and 61 to one line, and remove 63 and 65, and it should be good.